src/pages/documentation/mobile-core/release-notes.md

```diff
@@ -1,5 +1,16 @@
 # Release Notes
 
+## January 30, 2021
+
```

Comment:
Would you be willing to add the following content as an entry for Core release notes 
(there is one more item for Core related to Messaging/UIServices. I will provide it after consulting with @rymorale ) : 
```
## February 9, 2023

### Android Mobile Core 2.0.1

* Fixed issue causing large delays during configuration download retries.
* Fixed issue causing backdrop opacity being set incorrectly in `MessageSettings`.
* Fixed issue with javadoc generation.
* Improved resource handling after network connection attempts.
* Improved database handling during fresh installation scenarios.

### Android Mobile Signal 2.0.1

* Fixed issue with javadoc generation.

### Android Mobile Lifecycle 2.0.1

* Fixed issue with javadoc generation.

### Android Mobile Identity 2.0.1

* Fixed issue with javadoc generation.

```